code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -582,7 +579,8 @@
 
 /obj/item/pizzabox/pizza_bomb/Initialize(mapload)
 	correct_wire = pick(wires)
-	box_tag = "classic pepperoni"
+	var/obj/item/pizzabox/mimic_box = pick(subtypesof(/obj/item/pizzabox) - /obj/item/pizzabox/pizza_bomb)
```

Comment:
Also we want this, to prevent it from picking autoarm boxes, and any other subtypes added in the future
```suggestion
	var/obj/item/pizzabox/mimic_box = pick(subtypesof(/obj/item/pizzabox) - subtypesof(/obj/item/pizzabox/pizza_bomb))
```

Reply:
We're both wrong, this should be typesof